src/pages/pupils/programmes/[programmeSlug]/units.tsx

```diff
@@ -55,9 +58,9 @@ const PupilUnitListingPage = ({
         }}
       >
         <PupilViewsUnitListing
-          units={curriculumData}
-          programmeFields={programmeFields}
-          programmeSlug={programmeSlug}
```

Comment:
PupilViewsUnitListing now takes the data in it's fully formatted state, rather than it needing to process the data.